src/conn/conn_api.c

```diff
@@ -9,6 +9,7 @@
 #include "wt_internal.h"
 
 static int __conn_statistics_config(WT_SESSION_IMPL *, const char *[]);
+static int __conn_verbose_cmd(WT_SESSION_IMPL *, const char *);
```

Comment:
There has been another case recently where verbose has been used to expose more information to MongoDB, which is ``recovery_progress`` - that is different to these options, since it's a general flag setting rather than a single request for information.